infra.bs

```diff
@@ -630,6 +630,13 @@ out-of-bounds, except when used with <a for=list>exists</a>.
 "<code>b</code>", "<code>c</code>", "<code>a</code>" ». Then |example|[1] is the <a>string</a>
 "<code>b</code>".
 
+<p>Additionally, a literal syntax can be used to express a <a>list</a> of integers: the range
+<var>n</var> to <var>m</var>, inclusive, creates a new <a>list</a> whose <a for=list>items</a> are
+all integers in the range <var>n</var> to <var>m</var>, inclusive.
```

Comment:
It would work if do `s/in the range/represent the range/`, or so, but is a bit circular as a definition.

To clarify the problem: the number 1 is **in the range** 1 to 4, so a list consisting of a single item 1 has all its items in the range 1 to 4.